app/Helpers/Helper.php

```diff
@@ -116,11 +116,11 @@ public static function statusTypeList()
 
     public static function managerList()
     {
-        $manager_list = array('' => '') + User::select(DB::raw('concat(last_name,", ",first_name," (",username,")") as full_name, id'))
-        ->whereNull('deleted_at', 'and')
+        $manager_list = User::where('deleted_at', '=', null)
```

Comment:
We lost username. Oops, I can fix that.  I'm also unsure, however, if this ever did what was desired? Unless I'm misreading it, did the original managersList ever scope to only people who can be managers? (Or is this something to be implemented in the future).